### PR TITLE
Split up aplication layout

### PR DESCRIPTION
**Pull request by dohoonk**
Delete this branch once merged

### CODE REVIEW

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+<% if notice %>
+  <div class="alert alert-success">
+    <%= notice %>
+    <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+
+<% elsif alert  %>
```
Comment (zibs) [inline app/views/layouts/_header.html.erb:19]: Does elsif require an else or will this still work?

**Comment by zibs** [comment]
LGTM +1

File: app/views/layouts/_header.html.erb (hunk 1 above)
Comment (samanthaming) [inline app/views/layouts/_header.html.erb:19]: tam did it this way, just following the mighty one...

Comment (zibs) [inline app/views/layouts/_header.html.erb:19]: +1 i think he knows what he's doing...

**Pull request merged**